modules/exploits/linux/http/nuuo_nvrmini_upgrade_rce.rb

```diff
@@ -0,0 +1,80 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => 'NUUO NVRmini - upgrade_handle.php Remote Command Execution',
+      'Description'    => %q{
+        NUUO NVRmini  ip camera web application. The upgrade_handle.php file is affected by the remote command execution vulnerability.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'Berk Dusunur <@berkdusunur>', 
+          
+          
+          
+          'numan turle <@numanturle>',
+        ],
+      'References'     =>
+        [
+          ['URL', 'https://www.berkdusunur.net/2018/11/development-of-metasploit-module-after.html'],
+          ['CVE', '2018-14933'],
+          ['EDB', '45070']
+        ],
+      'Privileged'     => false,
+      'Payload'        =>
+        {
+          'DisableNops' => true
+        },
+      'Platform'       => %w{ unix win },
```

Comment:
`unix win` ? What about Linux `linux` ?

The module is in the `modules/exploits/linux/http/` directory.

If Windows is also supported, move the module here:

`modules/exploits/multi/http/nuuo_nvrmini_upgrade_rce.rb`

Reply:
okay ı updating